ISSUE_TEMPLATE/RFC_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+---
+name: 🎆 RFC
+about: Request for comments
+title: '[RFC]'
+labels: enhancement
+assignees: ''
+---
+**What are your requirements? What kind of business use-case are you trying to solve?**
+A clear and concise description of the business problem, e.g. _My customers are asking for [...]_
+
+**What is the problem? What is preventing you from meeting the requirements?**
+Describe what impact this problem has for lack of a solution.
+
+**What are you proposing? What do you suggest we do to solve the problem or improve the existing situation?**
+Describe your proposed solution. It's OK if you don't have one.
+
+**What are your assumptions or prerequisites?**
+Describe any assumptions you may be making that would limit the scope of this proposal.
+
+**What are remaining open questions?**
+List questions that may need to be answered before proceeding with an implementation.
+
```

Comment:
Should we add an optional section "What would you like people to specifically comment on?"

Reply:
Adding!